meson.build

```diff
@@ -453,6 +453,9 @@ if build_standalone
                             required: get_option('plugin_flashrom').disable_auto_if(host_machine.system() != 'linux'))
 endif
 
+libfdt = cc.find_library('fdt', required: get_option('plugin_vbe'))
+libz = cc.find_library('z', required: get_option('plugin_vbe'))
```

Comment:
For CI coverage add `zlib1g-dev` to `contrib/ci/dependencies.xml` (for Ubuntu/Debian).

And don't you want to be using `dependency('zlib', required: get_option('plugin_vbe')`?  Then you'll get all the CFLAGS and LDFLAGS associated with the pkgconfig for this library.

Reply:
done